ludwig/profiling/proto/dataset_profile_pb2.py

```diff
@@ -18,9 +18,9 @@
   name='ludwig/profiling/proto/dataset_profile.proto',
```

Comment:
Seems like this file ought to be in a `python` folder for consistency with having a `go` folder.

Reply:
Same with the `whylogs_messages_pb2.py` file.